ReleaseNotes.md

```diff
@@ -1,5 +1,24 @@
 # Release Notes
 
+## 2.0.0
+
+- Add support for `net462`, `net7.0` and `net8.0`
+- **[BREAKING CHANGE]** Drop support for `net461`, `netcoreapp3.1` and `net5.0`, all of which are EOL
```

Comment:
I appreciate we're making a breaking change to remove some of the EOL packages, but I'd like to see the targeting frameworks align with Microsoft's `System.Text.Json`, which currently supports `netstandard2.0`, `net6.0`, `net7.0`, `net8.0` and `net462`. 

Because I accidentally added `netstandard2.1` from the very beginning (for the purpose of using nullable reference etc), we also need to keep that and if it really bothers, we can consider it in another PR to consolidate it.

We won't be able to remove `net462` from the targeting framework list even though it has been EOL because that will force our .NET Framework users to recompile against `netstandard2.0` which is not fully supported until .NET Framework 4.7.2.

Reply:
This is a fair point. I thought it might be a (mostly) transparent change, but you're right that there could be some edge cases where the change in support could break things for some users.